projects/archiver/main.ts

```diff
@@ -15,26 +15,30 @@ import { unnest } from 'ramda'
 import { imageSizes, issueDir, ImageSize, Image } from '../common/src/index'
 import { bucket } from './s3'
 import { generateIndex } from './summary'
+interface Record {
+    s3: { bucket: { name: string }; object: { key: string } }
+} //partial of https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html
 
-const fetch = async (id: string): Promise<void> => {
+const fetch = async (source: string, id: string): Promise<void> => {
```

Comment:
My first comment would be can we document `source` somewhere? It is all pervasive and yet is not really clear what it does for a first time reader

Reply:
ok, I'll write up paths in an adr as a serperate pr

Reply:
That sounds like a plan 🆗